pylint/checkers/refactoring/refactoring_checker.py

```diff
@@ -1381,6 +1381,13 @@ def _is_node_return_ended(self, node: astroid.node_classes.NodeNG) -> bool:
             return any(
                 self._is_node_return_ended(_child) for _child in all_but_handler
             ) and all(self._is_node_return_ended(_child) for _child in handlers)
+        if (
+            isinstance(node, astroid.Assert)
+            and isinstance(node.test, astroid.Const)
+            and not node.test.value
+        ):
+            #  consider assert False as a return node
```

Comment:
```suggestion
            # consider assert False as a return node
```

Reply:
https://github.com/psf/black/issues/2091 😄

Reply:
@hippo91 just use ``setxkbmap -option "nbsp:none"`` in the meantime :p  https://stackoverflow.com/questions/31383074/how-to-disable-non-breaking-space-with-altgrspace?noredirect=1&lq=1

Reply:
Thanks to both of you!